S2VX.Game/Editor/Containers/InfoBar.cs

```diff
@@ -0,0 +1,70 @@
+﻿using osu.Framework.Allocation;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osuTK.Graphics;
+using S2VX.Game.Editor.UserInterface;
+using SixLabors.ImageSharp.Processing;
+
+namespace S2VX.Game.Editor.Containers {
+    public class InfoBar : CompositeDrawable {
+        [Resolved]
+        private S2VXEditor Editor { get; set; }
+
+        private SpriteText TxtTool { get; set; } = new SpriteText {
```

Comment:
If TxtTool is never set again, then you can remove the `set;`. This will essentially make the property readonly. You can apply this to other similar properties.